### ISSUE TITLE
Tabview with custom html in the header

### ISSUE THREAD

**Issue by bevrard** [primefaces/primeng#2462]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[X ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
For now the title of a tabview is a simple text.

**Expected behavior**
Would it be possible to add custom html to have dynamic content?

**What is the motivation / use case for changing the behavior?**
The objective is to implement a case with notifications on the tab title like on the following picture:
![image001](https://cloud.githubusercontent.com/assets/8151690/24763083/9d7e8d8a-1af0-11e7-80a9-59724f885d98.png)

Thanks in advance

**Comment by mikosu3** [comment]
i want it too.

**Comment by cagataycivici** [comment]
We'll see if we can do it, I believe we can do so with templating.

**Comment by nareshurs** [comment]
+1
We are in need of this feature too. I have been trying to customize however seems no way. Is there any workaround till this feature is implemented?

**Comment by alexschilpp** [comment]
+1

for the accordion panel you can simply create a custom header by adding `p-header` to the accordion tab

```
<p-accordionTab>
    <p-header>
        Header Content
    </p-header>
    Body Content
</p-accordionTab>
```

would be great if you could make that consistent for other header components too

**Comment by JacobMarshell** [comment]
I need this feature as well for my current project.

**Comment by nareshurs** [comment]
#cagataycivici, Is there a plan to implement this in nearest future,  We might look for other frameworks as there is no way to achieve this in priming.

**Comment by bevrard** [comment]
@nareshurs your choice on the framework is based on one component? you could implement your own component first and intergrate it with primeng when the functionality will be available.

**Comment by nareshurs** [comment]
@bevrard ,  At this point of time we have tight schedule which stopping me from creating custom component.  I was just at least expecting this issue tagged to one of priming future milestones if they are serious on this.  However I found a workaround for now by altering left icon with css.

**Comment by vangalder** [comment]
I'm also tasked with coming up with badge functionality for the menubar.

1. Are there currently any plans to add that functionality in the future?
2. Is there a way I can pass an identifier in for a top-level menu item?

**Comment by malbarmawi** [comment]
Its working right now ( primeng: 5.2.3) why this issue still open ?

**Comment by zadkiel87** [comment]
Any news on this subject ?

I need this feature as well for my current project.

**Comment by gbourant** [comment]
This feature works on premium templates , will it be available for the free ones ?

**Comment by upondeep** [comment]
+1 like a badge

**Comment by Ne-Ne** [comment]
I ideally want to style the headers with a unique style, at the minute you are limited to basic text and an icon. I believe It would allow for the creation of cleaner code.

Is there any update on this?

**Comment by dkvwu** [comment]
Is there any update?

**Comment by danmcmyler** [comment]
I was scrolling to the bottom expecting/hoping for this to be resolved. Seems like a popular request.

**Comment by danielsmeyer** [comment]
I am having trouble even implementing headerStyleClass.

**Comment by cagataycivici** [comment]
Available for 6.1.3 which is due later today.
